palvelutarjotin/settings.py

```diff
@@ -84,6 +84,10 @@
     TOKEN_AUTH_API_AUTHORIZATION_FIELD=(str, "authorization.permissions.scopes"),
     HELUSERS_BACK_CHANNEL_LOGOUT_ENABLED=(bool, False),
     HELUSERS_USER_MIGRATE_ENABLED=(bool, False),
+    HELUSERS_PASSWORD_LOGIN_DISABLED=(bool, False),
```

Comment:
I think disabling password login by default would be safer here in order to not allow it by accident in production, as in production it should be disabled:
```suggestion
    HELUSERS_PASSWORD_LOGIN_DISABLED=(bool, True),
```

Reply:
Okay, I can do that. I still want to mention that our team have earlier implemented this to Kummilapset and Notification-Service and they had this same setting `HELUSERS_PASSWORD_LOGIN_DISABLED=(bool, False),`, meaning it's set to _not be disabled_ by default.